test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -245,11 +245,6 @@ public static IEnumerable<object[]> LargeUploadData
         private static Socket CreateSocket(int port)
         {
             var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-
-            // Timeouts large enough to prevent false positives, but small enough to fail quickly.
-            socket.SendTimeout = 10 * 1000;
```

Comment:
Will probably be a separate change since it's not specific to the changes introduced here, but yeah I'll want your feedback on it.